File: game_server/ServerProtocol/src/CommonEnum/EPlatParamType.java
package CommonEnum;

/*********
 * 平台参数类型
 **/
public enum EPlatParamType {
	CLIENT_VERSION, //0 ==== 客户端资源版本
	HOT_REF_URL_BASE, //1 ==== 热更配表的CDN地址
	;
public static final EPlatParamType[]  EPlatParamType_Values = EPlatParamType.values();
public static final int EPlatParamType_Length = EPlatParamType_Values.length;
public static EPlatParamType EPlatParamType_FromInt(int _ivalue) {
	if(_ivalue < 0 || _ivalue >= EPlatParamType_Length){ return null; }
	return EPlatParamType_Values[_ivalue];
}
}

File: game_client/Assets/Scripts/ClientProtocol/CommonEnum/EPlatParamType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ALBasicProtocolPack;
using UnityEngine;


namespace CommonEnum
{

/// <summary>
/// 平台参数类型
/// </summary>
public enum EPlatParamType {
	[InspectorName("CLIENT_VERSION - idx[0] - 客户端资源版本")]
	CLIENT_VERSION, //0 ==== 客户端资源版本
	[InspectorName("HOT_REF_URL_BASE - idx[1] - 热更配表的CDN地址")]
	HOT_REF_URL_BASE, //1 ==== 热更配表的CDN地址
}

public class EPlatParamTypeComparer : IEqualityComparer<EPlatParamType>{
	public bool Equals(EPlatParamType x, EPlatParamType y) { return x == y; }
	public int GetHashCode(EPlatParamType obj) { return obj.GetHashCode(); }
	public static int g_iEnumCount = 2;
}
}

